.github/workflows/_test_python_source.yml

```diff
@@ -21,9 +21,6 @@ on:
 
 jobs:
   run:
-    permissions:
```

Comment:
What is this change for?

Reply:
This was merged by mistake and was causing issues for CI.